Formula/f/flight_sql.rb

```diff
@@ -0,0 +1,28 @@
+class FlightSql < Formula
+  desc "Apache Arrow Flight SQL Server - with DuckDB and SQLite back-ends"
+  homepage "https://arrow.apache.org/docs/format/FlightSql.html"
+  url "https://github.com/voltrondata/flight-sql-server-example.git",
+      tag:      "v1.1.19",
+      revision: "96f9831a901c4660c9cfb6cf299a682ef9614bb2"
+  license "Apache-2.0"
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "ninja" => :build
+  depends_on "openssl@3" => :build
+  depends_on xcode: :build
+  depends_on :macos
```

Comment:
Upstream builds for Linux also, why is this necessary?

Reply:
I got compiler errors on Mac for some reason using the default compilers that brew uses.  I was targeting Mac users with this formula, but wouldn't be opposed to building for both if I can get past the Mac compiler errors.  My upstream CI pipeline builds for MacOS fine, so I'm a little confused.

Reply:
if I remove the `depends_on: macos` line, would that be sufficient for supporting Linux as well?  I presume the Xcode dependency is ignored on Linux...

Reply:
I am disabling Linux build for now - I am getting cmake errors that are very difficult to understand.  This doesn't happen for brew Mac builds, and doesn't happen in my CI/CD pipeline where I build for Linux.  I'm targeting macOS users for this anyway - if ok, I would like to skip Linux for now.